### ISSUE TITLE
Accessibility Bug -  .NET Editor disappears at 400% zoom.  

### ISSUE THREAD

**Issue by LadyNaggaga** [MicrosoftDocs/feedback#2529]
Video of the bug is include here  [.NET Editor disappears at 400% zoom.](https://github.com/dotnet/try/files/4207845/787_A11y_.NETOnlineREPL_.NET.Editor_Reflow.pptx)
**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

#A11yTCS;#A11ySev1;#A11y_.NETOnlineREPL;#A11yWCAG2.1;#.NET Online REPL;#Benchmarks;#DesktopWeb;#Win10;#WCAG1.4.10;#Accessibility;#.NET Online REPL;#Reflow;#ChromiumEdge;

**​Repro Steps:​**
1.Change display resolution to 1280*1024.
2.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
3.Tab till "Enter focus Mode" button and press enter.
4.Zoom out to 400%
5.Check out any controls and information are missing or not.

**Actual Result:​**
When scenarios is enlarged to 400% and click on "Enter focus Mode" button and we can see that content overlaps and there is no scroll bar present.
In Reflow, unable to see content in the scenario and also controls are missing.

**Expected Result:​**
In Reflow, one scroll bar should be provided and all the control must be accessible to the user and no information in the scenario and no controls should get disappeared.

**User Impact:​**
Low vision Users will find difficult in accessing the controls and the content(information) if the content and the controls gets disappears.

**Attachment:**
[787_A11y_.NETOnlineREPL_.NET Editor_Reflow.pptx](https://github.com/dotnet/try/files/4207845/787_A11y_.NETOnlineREPL_.NET.Editor_Reflow.pptx)

**Comment by AbhitejJohn** [comment]
Here's a comment from a recent test pass where this still repros: https://github.com/dotnet/try/issues/787#issuecomment-1369736413

**Comment by fsteffi** [comment]
this bug has been opened as part of https://github.com/dotnet/try/issues/787 issue is related to docs.microsoft.com
